diff --git a//root/workspace/data_collector/copy.java b//root/workspace/data_collector/copy.java
index 7605567..58bf8fa 100644
--- a//root/workspace/data_collector/copy.java
+++ b//root/workspace/data_collector/copy.java
@@ -79,6 +79,9 @@ org.openrewrite.java.cleanup.HideUtilityClassConstructor, org.openrewrite.java.cleanup.OperatorWrap
      * definition statement.
      */
     Node getRValue();
+
+      private Definition <error>() {
+      }
   }
 
   /**
@@ -296,8 +299,8 @@
     VarDefinition(Node node) {
       Preconditions.checkArgument(NodeUtil.isVarDeclaration(node));
       Preconditions.checkArgument(node.hasChildren(),
-          "VAR Declaration of " + node.getString() +
-          "should be assigned a value.");
+          "VAR Declaration of " + node.getString()
+          + "should be assigned a value.");
       name = node;
     }
 
@@ -322,4 +325,7 @@
       return name.getFirstChild();
     }
   }
+
+    private DefinitionsRemover() {
+    }
 }
